packages/incremental-merkle-tree.sol/contracts/IncrementalBinaryTree.sol

```diff
@@ -48,6 +120,17 @@ library IncrementalBinaryTree {
         self.root = zero;
     }
 
+    function initDefaultZeroes(IncrementalTreeData storage self, uint256 depth) public {
```

Comment:
What about `initWithDefaultZeroes`? `initDefaultZeroes` suggests you are initializing the default zeroes (which are actually constants), and not the tree.